secboot/keymgr/keymgr_luks2.go

```diff
@@ -119,11 +117,11 @@ func AddRecoveryKeyToLUKSDeviceUsingKey(recoveryKey keys.RecoveryKey, currKey ke
 		Slot:       recoveryKeySlot,
 	}
 	if err := luks2.AddKey(dev, currKey, recoveryKey[:], &options); err != nil {
-		return fmt.Errorf("cannot add key: %w", err)
+		return fmt.Errorf("cannot add key: %v", err)
```

Comment:
AFAICT, this is fine but, just out of curiosity, what's the reason to not wrap the error?